bot/cogs/tags.py

```diff
@@ -146,7 +146,7 @@ def command_on_cooldown(self, ctx: Context, tag_name: str) -> bool:
                     title='Did you mean ...',
                     description='\n'.join(tag['title'] for tag in founds[:10])
                 ))
-                return True
+                return False
```

Comment:
This should still return True otherwise the bot will send a command suggestion after the tag suggestions, which is not desirable.

Reply:
Yeah, I will add a parameter in that case

Reply:
I concur that this should return a True, otherwise this is possible

![image](https://user-images.githubusercontent.com/19921765/75602909-29564700-5afc-11ea-8a21-4b7f1cedeec8.png)